lightningd/opening_control.c

```diff
@@ -660,8 +660,10 @@ static void channel_config(struct lightningd *ld,
 {
 	/* FIXME: depend on feerate. */
 	*max_to_self_delay = ld->config.locktime_max;
-	/* This is 1c at $1000/BTC */
-	*min_effective_htlc_capacity = AMOUNT_MSAT(1000000);
+
+	/* Take minimal effective capacity from config min_capacity_sat */
+	u64 min_capacity_msat = ld->config.min_capacity_sat * 1000;
```

Comment:
As a goal, we're moving away from having `* 1000` inline for sat/msat transformations. A better way to handle this would be to have a helper, i.e. `amount_msat_from_satval(...)`

Reply:
added `amount_msat_from_sat_u32`  for this purpose